lxd/db/generate/db/method.go

```diff
@@ -193,11 +193,7 @@ func (m *Method) getMany(buf *file.Buffer) error {
 	buf.N()
 	buf.L("// Select.")
 	buf.L("err := query.SelectObjects(stmt, dest, args...)")
-	buf.L("if err != nil {")
-	buf.L("        return nil, errors.Wrap(err, \"Failed to fetch %s\")", lex.Plural(m.entity))
-	buf.L("}")
-	buf.N()
-
+	m.ifErrNotNil(buf, "nil", fmt.Sprintf("errors.Wrap(err, \"Failed to fetch %s\")", lex.Plural(m.entity)))
```

Comment:
@masnax looks like `errors.Wrap` is still being used here.

Reply:
@masnax still seeing a bunch of errors.Wrap

Reply:
@stgraber I didn't remove them everywhere in the generator, just in `lxd/db/generate/db/method.go`. Since it's an extra commit on this PR anyway, I can just remove `errors.Wrap` everywhere in the generator as one commit.